arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -211,7 +211,7 @@ std::string const RestReplicationHandler::LoggerTickRanges =
 std::string const RestReplicationHandler::LoggerFirstTick = "logger-first-tick";
 std::string const RestReplicationHandler::LoggerFollow = "logger-follow";
 std::string const RestReplicationHandler::OpenTransactions =
-    "determiconst ne-open-transactions";
+    "determine-open-transactions";
```

Comment:
why was this not caught in the PR tests before ?

Reply:
testing.js just ignored the test results.

Reply:
that was fixed by @dothebart on Fri

Reply:
Because `replication_ongoing_32` and `replication_ongoing_frompresent_32` were not included in PRs.